muxdb/muxdb.go

```diff
@@ -62,6 +63,21 @@ type MuxDB struct {
 	trieBackend *backend
 }
 
+func collectCompactionMetrics(ldb *leveldb.DB) {
+	ticker := time.NewTicker(10 * time.Second)
+	defer ticker.Stop()
+
+	for range ticker.C {
```

Comment:
Should this process also stop when the muxdb is `Close()` ?

Reply:
Modification done here https://github.com/vechain/thor/pull/936/commits/916c017ca6447069dedd15fb96f7383b2bce1f1c to allow that, please have a look, thanks.